core/command_queue_mt.cpp

```diff
@@ -107,6 +107,7 @@ CommandQueueMT::CommandQueueMT(bool p_sync) {
 	write_ptr = 0;
 	dealloc_ptr = 0;
 	mutex = Mutex::create();
+	command_mem = (uint8_t *)malloc(COMMAND_MEM_SIZE);
```

Comment:
Before anyone complains, malloc() here guarantees that the array itself is 64bit aligned :)